### PR TITLE
Create image of Chromium

### PR DESCRIPTION
**Pull request by tanin47**
* Ansible command to create an image.
* The CI will just use the image to create a disk.

We don't clone chromium src on CI anymore. That should speed things up and make the CI more reliable.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,9 +60,22 @@ Develop
 4. Run the tool binary: `./pants run tool:clusterfuzz-ci -- reproduce -h`
 
 
+Create a CI image
+------------------
+
+From the `ci` directory, perform the below steps:
+
+1. Modify `image.yml` and increment the version of `image_name` in `group_vars/all`
```
Comment (inferno-chromium) [inline README.md]: Add period.

Comment (inferno-chromium) [inline README.md]: Also, can we remove the CI stuff from main README. Noone except us cares for this and should be in a seperate document in docs/ folder.

File: ci/image.yml [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+        group: clusterfuzz
+        mode: 0777
+    - name: Fetch Chromium
+      command: "fetch --nohooks --force --no-history chromium"
+      args:
+        chdir: "{{ home }}/chromium"
+    - name: Accept mscorefonts EULA
+      command: bash -c "echo ttf-mscorefonts-installer msttcorefonts/accepted-mscorefonts-eula select true | sudo debconf-set-selections"
+    - name: Run install-build-deps.sh
+      command: "build/install-build-deps.sh --no-prompt"
```
Comment (inferno-chromium) [inline ci/image.yml]: Can you add --chromeos-fonts and --lib32

File: ci/image.yml [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+      args:
+        chdir: "{{ home }}/chromium/src"
+    - name: Run install-build-deps.sh (32-bit)
+      command: "build/install-build-deps.sh --lib32 --syms --no-prompt"
```
Comment (inferno-chromium) [inline ci/image.yml]: This could be just done as part of previous command.

File: ci/group_vars/all [hunk 4]
```diff
@@ -0,0 +1,11 @@
+---
+
+# Increment the number when modifying the image.
+image_name: chromium-image-1
+
+project_id: 981641712411
+zone: us-central1-a
+
+service_account_email: ci-round-2@clusterfuzz-tools.iam.gserviceaccount.com
```
Comment (inferno-chromium) [inline ci/group_vars/all]: can we rename this account to just ci@ ?

File: ci/group_vars/all [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+credentials_file: /auto/FuzzInfrastructure/clusterfuzz-tools/clusterfuzz-tools-ci-credentials.json
```
Comment (inferno-chromium) [inline ci/group_vars/all]: s/clusterfuzz-tools-ci-credentials.json/ci-credentials.json

File: ci/image.yml (hunk 2 above)
Comment (tanin47) [inline ci/image.yml]: I didn't know that. Thank you!

File: ci/image.yml (hunk 3 above)
Comment (tanin47) [inline ci/image.yml]: I didn't know that. Thank you!

File: ci/group_vars/all (hunk 4 above)
Comment (tanin47) [inline ci/group_vars/all]: Not right now. It's tied to our service account's email. We can roll the credentials later.

File: ci/group_vars/all (hunk 5 above)
Comment (tanin47) [inline ci/group_vars/all]: Let's do it separately.

**Pull request merged**